### DISCUSSION TITLE
axaml can work as xaml in wpf?

### DISCUSSION THREAD

**Discussion by chancezheng** [AvaloniaUI/Avalonia#5459] [Q&A]
When I start the avalonia app, then alter the content in axaml, the app was not autoupdate, Can it work as wpf, so we can debug easily

**Comment by FoggyFinder** [comment]
Do you mean "Hot reload"?
If so see [XAML hot reload support #3266](https://github.com/AvaloniaUI/Avalonia/issues/3266) for details

**Reply by FrankenApps** [reply]
Well he asks if it's possible to use Avalonia XAML files in WPF in order to use WPFs hot reload for quicker debugging.
Apart from some exceptions the answer is no, because Controls have different names and different properties in general.

**Reply by chancezheng** [reply]
Thanks,  When Can avalonia support XAML hot-reload ?
